backend/start_server.py

```
@@ -51,10 +56,25 @@ def check_model():
         print(f"❌ Error checking model: {e}")
         return False
 
+
 def start_api_server():
```

Ruff (0.12.2)

Missing return type annotation for public function `start_api_server`

Add return type annotation: `None`

(ANN201)